dev/automated_tests/test_smoke_test/pending_timer_fail_test.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  failingPendingTimerTest();
+}
+
+void failingPendingTimerTest() {
+  testWidgets('flutter_test pending timer - negative', (WidgetTester tester) async {
+    Timer(const Duration(seconds: 1), () {});
```

Comment:
I'm not sure if this makes a difference but can we make the duration longer, like a few minutes?  Just in case this runs on a slow CI machine and actually takes a second to make it all the way through this code (it seems pretty unlikely, but far less likely if the timer is for a few minutes or hours or something).

Reply:
I'll change it to 10 minutes.

(BTW, I *meant* to squash #37981 but picked the wrong choice by accident.  Sorry about that.)